File: app/files/stack.js
class Stack extends React.Component {
  render() {
    return (
      <div className={'stack'}>
        <h2>{this.props.stack.name}</h2>
        <div className={'services'}>
          {this.props.stack.services.map(service => (
            <Service
              key={service.name}
              service={service}
              manifest={this.props.manifest}
            />
          ))}
        </div>
      </div>
    );
  }
}

File: shared/skep/docker/stack.py
class Stack:
    def __init__(self, name, service_list):
        self.name = name
        self.service_list = service_list

    def services(self):
        return [x.attrs() for x in self.service_list]

    def serializable(self):
        return {
            "name": self.name,
            "services": self.services()
        }
